infra/bridge/src/controllers/promotions.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
+
+const express = require('express')
+const router = express.Router()
+const { getAsync } = require('../utils/redis')
+
+const Attestation = require('./../models/attestation')
+
+// TODO: Add request body validation
+router.post('/verify', async (req, res) => {
+  const { type, socialNetwork, identity } = req.body
+
+  const attestation = await Attestation.findOne({
+    ethAddress: identity,
+    method: socialNetwork
+  })
+
+  if (!attestation) {
+    return res.status(400).send({
+      success: false,
+      errors: [`Attestation missing`]
+    })
+  }
+
+  const redisKey = `${socialNetwork.toLowerCase()}/${type.toLowerCase()}/${
+    attestation.value
+  }`
+  let tries = 0
+  const interval = setInterval(async () => {
+    const event = await getAsync(redisKey)
+
+    if (event) {
+      clearInterval(interval)
+      // TODO: Process event and Push to growth events
+      return res.status(200).send({
+        success: true
+      })
+    } else {
+      tries++
+    }
+
+    if (tries >= 300) {
+      // Request will timeout after 1000ms * 300 === 300s or 5mins
+      clearInterval(interval)
+      return res.status(200).status({
```

Comment:
nit: I would log.error here. so that we can know for looking at logs if verifications are failing.
And I would log.info when verification is successful.